.teamcity/tests/buildTypes/E2ETests.kt

```diff
@@ -15,35 +14,30 @@ object E2ETests : BuildType({
     root(vcsRoots.KotlinLangOrg)
   }
 
-  steps {
-    script {
-      scriptContent = "./scripts/test/up.sh"
-    }
+    dependencies {
+        artifacts(BuildReferenceDocs) {
+            cleanDestination = true
```

Comment:
Maybe we can reuse there the last successful build to speed up the process?
We do it in a different build: 
```
buildRule = lastSuccessful(branch = "+:<default>")
```

Reply:
I'm not sure. We want to test branch before release, no? I think it's nice build by chain.

Reply:
So we need to decide something here.
For me it is also sounds more reliable to build needed artifacts explicitly before tests run (this is to leave things as it is now)